Samples/PowerShell/functions/func_ModernDashboards.ps1

```diff
@@ -0,0 +1,489 @@
+    PS C:\> Export-ModernDashboard -SwisConnection $SwisConnection -DashboardId 9 -IncludeId
+
+    This exports the Modern Dashboard with ID 9 to the current folder with the naming format "9_<Dashboard Name>.json"
+.NOTES
+    Author:  Kevin M. Sparenberg (https://thwack.solarwinds.com/members/kmsigma)
+    Version: 0.9
+    Last Updated: 2021-06-28
+    Validated: Orion Platform 2020.2.5
+
+    TBD List:
+        * Add/incoporate the check for valid/invalid file names from Utilities
+        * -PassThru [switch]
+            Mimic a -PassThru parameter that just returns the JSON data to the pipeline
+            -AsPsObject [switch] (child of -PassThru)
+                Allows for the raw Json to be returned as a PowerShell object
+#>
+function Export-ModernDashboard {
+    [CmdletBinding(
+        DefaultParameterSetName = 'Normal', 
+        SupportsShouldProcess = $true, 
+        PositionalBinding = $false,
+        HelpUri = 'https://documentation.solarwinds.com/en/success_center/orionplatform/content/core-fusion-dashboard-import-export.htm',
+        ConfirmImpact = 'Medium')]
+    [Alias()]
+    [OutputType([String])]
+    Param
+    (
+        # The connection to the SolarWinds Information Service
+        [Parameter(
+            Mandatory = $true, 
+            ValueFromPipeline = $true,
+            ValueFromPipelineByPropertyName = $true, 
+            ValueFromRemainingArguments = $false, 
+            Position = 0,
+            ParameterSetName = 'Normal')]
+        [ValidateNotNull()]
+        [ValidateNotNullOrEmpty()]
+        [Alias("Swis")] 
+        [SolarWinds.InformationService.Contract2.InfoServiceProxy]$SwisConnection,
+
+        # The dashboard Id we'll export
+        [Parameter(
+            ValueFromPipeline = $true,
+            ValueFromPipelineByPropertyName = $true,
+            Position = 1,
+            ParameterSetName = 'Normal')]
+        [AllowNull()]
+        [int32[]]$DashboardId,
+
+        # Specifies the path to the output file.
+        [Parameter(ParameterSetName = 'Normal')]
+        [AllowNull()]
+        [string]$OutputFolder = ( Get-Location ),
+
+        # Should we include system Dashboards?
+        [Parameter(ParameterSetName = 'Normal')]
+        [AllowNull()]
+        [switch]$IncludeSystem,
+
+        # Should we include the Dashboard ID number in the name.
+        [Parameter(ParameterSetName = 'Normal')]
+        [AllowNull()]
+        [switch]$IncludeId,
+
+        # Omits white space and indented formatting in the output string.
+        [Parameter(ParameterSetName = 'Normal')]
+        [AllowNull()]
+        [switch]$Compress,
+
+        # Overrides the read-only attribute and overwrites an existing read-only file. The Force parameter does not override security restrictions.
+        [Parameter(ParameterSetName = 'Normal')]
+        [AllowNull()]
+        [switch]$Force
+
+    )
+
+    Begin {
+        # if no dashboard ids are provided, assume we export the all, so get the list
```

Comment:
"the all" => "them all".  It would also be nice to capitalize "If".

Reply:
Yep - got too fast with my typing.